keyboards/handwired/scottokeebs/scotto34/info.json

```diff
@@ -0,0 +1,87 @@
+{
+    "manufacturer": "ScottoKeebs",
+    "keyboard_name": "Scotto34",
+    "maintainer": "joe-scotto",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": [
+            "GP3",
+            "GP2",
+            "GP7",
+            "GP8",
+            "GP13",
+            "GP16",
+            "GP17",
+            "GP18",
+            "GP19",
+            "GP20"
+        ],
+
+        "rows": ["GP0", "GP4", "GP9", "GP14"]
+    },
+    "processor": "RP2040",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0xFEED"
+    },
+    "layouts": {
+        "LAYOUT_ortho_3x10_4": {
```

Comment:
Entirely optional, but changing `LAYOUT_ortho_3x10_4` to `LAYOUT_split_3x5_2` will allow community layouts to automatically be assigned (namely miryoku).  
Would have to apply this change to all `keymap.c` files also.

Reply:
Switched in [ccb3b7a](https://github.com/qmk/qmk_firmware/pull/20943/commits/ccb3b7aed62119f720476da2ce67ec8dd0afc9a3).